src/server/utils/tracking.ts

```diff
@@ -17,6 +17,10 @@ const snap = `
 `
 
 export const allTracking = (nonce: string) => `
+<script nonce="${nonce}">
+  dataLayer = [];
```

Comment:
Is it more clear to write it out like:

```suggestion
  window.dataLayer = [];
```

Reply:
I kinda like to stick to the dev docs from google https://developers.google.com/tag-manager/devguide#adding-data-layer-variables-to-a-page :)